### PR TITLE
 add heading levels to care cards

### PR DESCRIPTION
**Pull request by Ross-Clark**
+ fix card names

### CODE REVIEW

File: wagtailnhsukfrontend/templates/wagtailnhsukfrontend/care_card.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="nhsuk-care-card nhsuk-care-card--{{ value.type }}">
   <div class="nhsuk-care-card__heading-container">
-    <h3 class="nhsuk-care-card__heading">
+    <h{{ heading_level }} class="nhsuk-care-card__heading">
```
Comment (AdamChrimes) [inline wagtailnhsukfrontend/templates/wagtailnhsukfrontend/care_card.html]: The closing tag for this heading level needs to be `<h{{ heading_level }}>`, not `</h3>`

Comment (Ross-Clark) [reply wagtailnhsukfrontend/templates/wagtailnhsukfrontend/care_card.html]: Thanks Adam

Comment (AdamChrimes) [reply wagtailnhsukfrontend/templates/wagtailnhsukfrontend/care_card.html]: @Ross-Clark not sure if you forgot to update the code or were just thanking me, but the code is still `</h3>` 😄

Comment (Ross-Clark) [reply wagtailnhsukfrontend/templates/wagtailnhsukfrontend/care_card.html]: I blame the wifi

**Pull request merged**